community/event-checklist.md

```diff
@@ -0,0 +1,96 @@
+
+- **Making the workshop interactive**: It is always a good thing to keep the workshop interactive. If the workshop is conducted offline, we can have members of KOSS help people personally with hands on during the workshop and clear doubts more personally. But during an online workshop, this is not possible. So for sake of interaction, the speakers can encourage the audience to share the screen to show the work, for kind of validation and it also helps the rest of the audience. Asking the audience to share their screen in case of difficulties should also be encouraged. However note that,**do not force the audience to share their screen for sake of interaction. It has be voluntary and audience should not feel uncomfortable through such actions.**
+
+# 2. Technicalities - Workshop content
+
+## 2.1 How to decide the Topic for the workshop
+
+- Make sure that the topic adds value to the community. The topic doesn't have to be trendy or in vogue.
+- Atleast a couple of members at KOSS should have enough practical experience to speak about the topic
+
+## 2.2 How to decide the content of the workshop
+
+Make sure that the content adds a practical value to the audience. It might be tempting to add bit of cool jargon or theoretically detailed stuff to impress. But it is not suggested. For every slide, ask - Is this something that adds practical value. Will knowing or not knowing a particular thing affect the practical understanding of the topic. For example, consider explaining the idea of Merkel Tree in git, it might not be necessary to include the explanation of Merkel tree. As someone getting started with git, it is more important to practically understand staging area, branching, commits, sending a PR, pushing and pulling. When we get into details like Merkel Tree, audience might loose track of the event and loose interest. However, for the curious audience, it is a good idea to include such things at the end of the workshop, and provide further resources to explore more. 
+
+During the core period of the workshop, make sure that they get a **practical hands on experience**(which is what they have come for, otherwise they could have learnt it on YouTube too!)
+
+Make sure to give enough time to cover the queries and concerns of the audience during the workshop.(without queries and interaction, the workshop would not differ from a YouTube Tutorial)
+
+## 2.3 How to organize or approach content:
+
+- **2.3.1 Introduction:** It is a good idea to tell the audience the following things: 
+
+  - What they should take away from the workshop. For example, if it is git workshop, convey that by the end of the workshop they will be comfortable to track their project using git and know how to send a Pull Request properly to a open source repository on Github.
+
+  - Give the audience about how the flow of workshop would be. For example...
```

Comment:
```suggestion
  - Give the audience about how the flow of workshop would be. For example - 
```
don't use ellipsis, although indians are more prone to use them... [link](https://www.reddit.com/r/indianpeoplefacebook/comments/5j0bn4/why_do_indians_overuse_ellipses_in_their_writing/)